lib/console/gateways/gateway.ex

```diff
@@ -18,16 +19,16 @@ defmodule Console.Gateways.Gateway do
     field :status, :string
 
     belongs_to :team, Team
+    belongs_to :hardware_identifier, HardwareIdentifier
```

Comment:
tried to put on delete: delete all here, would not compile =[

Reply:
ok